spec/requests/partner_users_requests_spec.rb

```diff
@@ -111,4 +111,23 @@
       end
     end
   end
+
+  describe "POST #reset_password" do
+    let!(:partner_user) do
+      UserInviteService.invite(
+        email: "meow@example.com",
+        name: "Meow Mix",
+        roles: [Role::PARTNER],
+        resource: partner
+      )
+    end
+
+    context "when the partner needs to reset a user's password" do
```

Comment:
Is this supposed to be when a bank needs to reset a partner user's password?

Reply:
Oh yes, that would be a more accurate description. 
Should I make the change and, if so, how would I add it to this pull request?

Reply:
Yes, please.  When you push your changes to github, they  will automatically come through to this pull request.

Reply:
Okay I've made that update!